File: CPP/Produk.cpp
// library
#include <string>

using namespace std;
class Produk;

// membuat kelas Produk
class Produk{ // ini redifinition produk error waktu gacukup buat benerin x.x
    // atribut yang diperlukan
    private:
        string id;
        
    public:
        string name;
        string brand;
        string price;
    
    public:
        // konstruktor kosong
        Produk() {
        }

        // konstruktor dengan parameter
        Produk(string id, string name, string brand, string price) {
            // set nilainya
            this->id = id;
            this->name = name;
            this->brand = brand;
            this->price = price;
        }

        // set & get id produk
        void setId(string id) {
            this->id = id;
        }

        string getId() {
            return this->id;
        }

        // set & get nama produk
        void setName(string name) {
            this->name = name;
        }

        string getName() {
            return this->name;
        }

        // set & get merek produk
        void setBrand(string brand) {
            this->brand = brand;
        }

        string getBrand() {
            return this->brand;
        }

        // set & get harga produk
        void setPrice(string price) {
            this->price = price;
        }

        string getPrice() {
            return this->price;
        }
        
        // deconstruct
        ~Produk() {
        }
};

File: Java/Produk.java
// membuat kelas produk
public class Produk extends Shirt {
    // atribut yang diperlukan
    private int id;
    private String name;
    private String brand;
    private String price;
    
    // konstruktor kosong
    Produk() {
    }

    // konstruktor dengan parameter
    Produk(int id, String name, String brand, String price) {
        this.id = id; 
        this.name = name;
        this.brand = brand;
        this.price = price;
    }
    
    // get & set id
    public void set_id(int id) {
        this.id = id;
    }

    public int get_id() {
        return this.id;
    }
    
    // get & set name
    public void set_name(String name) {
        this.name = name;
    }

    public String get_name() {
        return this.name;
    }

    // get & set brand
    public void set_brand(String brand) {
        this.brand = brand;
    }
    
    public String get_brand() {
        return this.brand;
    }

    // get & set price
    public void set_price(String price) {
        this.price = price;
    }
    
    public String get_price() {
        return this.price;
    }
}